index.html

```diff
@@ -0,0 +1,47 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+    <title>Mourid Portfolio</title>
+</head>
+<body>
+    <section>
+    <nav class="navbar">
+        <div class="container">
+            <div class="logo"><img src="img/My Logo.png" alt=""></img></div>
+        <div class="menu"><img src="img/Union.svg" alt=""></div>
+    </div>
+     </nav>
+    <header class="header">
+        <div class="container">
+                    <div class="textInfo">
+             <h1>I’m Mariane
+                Glad to see you!</h1>
+                
+                    <p>I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+                    </div>        
+    </div>
+    <p class="con">LET'S CONNECT</p>
+    <div class="social">     
```

Comment:
- [ ] Please , good job for adding the lets connect button but it would be better to put it inside a `a` tag or a `button` tag